File: Algorithms/C/Sorting Algo/Bubble Sort/bubbleSort.c
#include<stdio.h>
#include<stdlib.h>
void bubblesort(int [],int);
 int main()
 {
     int arr[100],i,n,j,temp,choice;
     printf("\n How many numbers you want to enter :");
     scanf("%d",&n);
     printf("\n Enter the numbers : ");
     for(i=0;i<n;i++)
     {
     scanf("%d",&arr[i]);
    }printf("\n Numbers before sorting : \n ");
     for(i=0;i<n;i++)
     {
         printf("%d   ",arr[i]);
     }
     bubblesort(arr,n);
 }
void bubblesort(int arr[],int n)
{
  long c, d, t;

  for (c = 0 ; c < n - 1; c++)
  {
    for (d = 0 ; d < n - c - 1; d++)
    {
      if (arr[d] > arr[d+1])
      {
        /* Swapping */

        t= arr[d];
        arr[d]   = arr[d+1];
        arr[d+1] = t;
      }
    }
  }
  printf("\n After bubble sorting : \n");
   for (c = 0; c < n; c++)
   {
    printf("%d   ", arr[c]);
   }
}

File: Data Structures/JavaScript/Sorting/bubbleSort.js
function bubbleSort(a)
{
    for(let i=0 ; i<a.length;i++){
        for(j=0;j<a.length-i-1;j++){
            if(a[j] > a[j+1]){
                temp=a[j]
                a[j]=a[j+1]
                a[j+1]=temp
            }
        }
    }
    return a
}
a=[8,4,5,2,6,1,0,9]
console.log(bubbleSort(a))